Eastern-Research-Group/mywaterway
# Feature/11 location page updates

## Related Issues:
* [HMW-11](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-11)

## Main Changes:
* Changed occurrences of 'station' to 'location'.
* Re-combined the added the org ID back to the site ID.
* Added a help tooltip to describe the site ID.
* Updated the download link to use only characteristic names and to use a POST request, while the WQP link uses only characteristic types.
* Updated the date slider to only show the year if the start and end dates are the same.
* Added some Cypress tests.

## Steps To Test:
1. Go to a monitoring location page, e.g. [http://localhost:3000/location/STORET/CBP_WQX/CBP_WQX-01651770/](http://localhost:3000/location/STORET/CBP_WQX/CBP_WQX-01651770/)
2. Select a characteristic from the upper-right table that only has one year of data, such as _Orthoposphate_
3. Confirm that only the year is shown where the date slider would be
4. Select the characteristic _Height, gage_
5. Confirm that on days with multiple measurements, both are shown on the graph and in the hover tooltip
6. In the download section, deselect a type, then select a single characteristic from that type
7. Confirm that the download link returns the correct number of results
